Userland/Libraries/LibGfx/ImageFormats/TIFFLoader.cpp

```diff
@@ -284,16 +284,16 @@ class TIFFLoadingContext {
             auto decoded_stream = make<BigEndianInputBitStream>(move(decoded_strip));
 
```

Comment:
Typos in commit message: "store" -> "stores" (2x), "relatively to start" -> "relative to the start"

Reply:
Ouch... thanks!

Reply:
It still says "relatively" instead of "relative"

Reply:
There you go